src/schemas/block.yaml

```
@@ -84,6 +87,11 @@ Block:
           type: array
           items:
             $ref: '#/components/schemas/TransactionSigned'
+    withdrawals:
+      title: Full withdrawals
```

This will be both full and partial withdrawals, won't it?  As such, I think this should be changed to just "Withdrawals".

---

I think that is correct.
```suggestion
      title: Withdrawals
```